src/Compilers/CSharp/Portable/Symbols/Source/SourceMethodSymbolWithAttributes.cs

```diff
@@ -345,13 +345,12 @@ internal sealed override CSharpAttributeData EarlyDecodeWellKnownAttribute(ref E
             return base.EarlyDecodeWellKnownAttribute(ref arguments);
         }
 
-        // PROTOTYPE(local-function-attributes): test local function with SkipLocalsInit
         public override bool AreLocalsZeroed
```

Comment:
>override [](start = 15, length = 8)

Can we seal this implementation?

Reply:
I think we need to override it in SynthesizedMethodBaseSymbol.

Reply:
I think we need to override it in SynthesizedMethodBaseSymbol.

---
In reply to: [371410335](https://github.com/dotnet/roslyn/pull/41183#discussion_r371410335) [](ancestors = 371410335)

Reply:
> I think we need to override it in SynthesizedMethodBaseSymbol.

SynthesizedMethodBaseSymbol inherits from SourceMemberMethodSymbol

---
In reply to: [371421097](https://github.com/dotnet/roslyn/pull/41183#discussion_r371421097) [](ancestors = 371421097,371410335)